test/module/iroha-cli/client_test.cpp

```diff
@@ -151,3 +152,125 @@ TEST_F(ClientTest, SendTxWhenStatelessInvalid) {
   std::cout << "Sending  json transaction to Iroha" << std::endl;
   ASSERT_EQ(client.sendTx(json_tx), iroha_cli::CliClient::NOT_VALID);
 }
+
+TEST_F(ClientTest, SendQueryWhenInvalidJson) {
+  iroha_cli::CliClient client(Ip, Port);
+  // Must not call stateful validation
+  EXPECT_CALL(svMock, validate(A<std::shared_ptr<const iroha::model::Query>>()))
+      .Times(0);
+  auto json_query =
+      "{\n"
+      "  \"creator_account_id\": \"test\", \n"
+      "  \"commands\":[{\n"
+      "  \"command_type\": \"AddPeer\",\n"
+      "    \"address\": \"localhost\",\n"
+      "    \"peer_key\": "
+      "\"2323232323232323232323232323232323232323232323232323232323232323\"\n"
+      "  }]\n"
+      "}";
+  std::cout << "Sending  json query to Iroha" << std::endl;
+  auto res = client.sendQuery(json_query);
+  ASSERT_TRUE(res.has_error_response());
+  ASSERT_EQ(res.error_response().reason(),
+            iroha::protocol::ErrorResponse::WRONG_FORMAT);
+}
+
+TEST_F(ClientTest, SendQueryWhenStatelessInvalid) {
+  iroha_cli::CliClient client(Ip, Port);
+  // Must not call stateful validation
+  EXPECT_CALL(svMock, validate(A<std::shared_ptr<const iroha::model::Query>>()))
+      .WillOnce(Return(false));
+  auto json_query =
```

Comment:
I guess, good solution for json string in test use raw strings (with `R` prefix).